### ISSUE TITLE
PDDatabaseStorage problem

### ISSUE THREAD

**Issue by rehandalal** [mozilla/django-product-details#62]
Some change in https://github.com/mozilla/django-product-details/commit/84124023f927fe9bd2a16340d241ff7024e34175 is causing an issue with using `PDDatabaseStorage` on https://github.com/mozilla/normandy.

When I set `PROD_DETAILS_STORAGE` to `'product_details.storage.PDDatabaseStorage'` I see the following:

```
Traceback (most recent call last):
  File "/home/rehandalal/.virtualenvs/normandy/lib/python3.5/site-packages/django/utils/autoreload.py", line 226, in wrapper
    fn(*args, **kwargs)
  File "/home/rehandalal/.virtualenvs/normandy/lib/python3.5/site-packages/django/core/management/commands/runserver.py", line 109, in inner_run
    autoreload.raise_last_exception()
  File "/home/rehandalal/.virtualenvs/normandy/lib/python3.5/site-packages/django/utils/autoreload.py", line 249, in raise_last_exception
    six.reraise(*_exception)
  File "/home/rehandalal/.virtualenvs/normandy/lib/python3.5/site-packages/django/utils/six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "/home/rehandalal/.virtualenvs/normandy/lib/python3.5/site-packages/django/utils/autoreload.py", line 226, in wrapper
    fn(*args, **kwargs)
  File "/home/rehandalal/.virtualenvs/normandy/lib/python3.5/site-packages/django/__init__.py", line 18, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/home/rehandalal/.virtualenvs/normandy/lib/python3.5/site-packages/django/apps/registry.py", line 85, in populate
    app_config = AppConfig.create(entry)
  File "/home/rehandalal/.virtualenvs/normandy/lib/python3.5/site-packages/django/apps/config.py", line 90, in create
    module = import_module(entry)
  File "/home/rehandalal/.virtualenvs/normandy/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 673, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 662, in exec_module
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "/home/rehandalal/.virtualenvs/normandy/lib/python3.5/site-packages/product_details/__init__.py", line 72, in <module>
    product_details = ProductDetails()
  File "/home/rehandalal/.virtualenvs/normandy/lib/python3.5/site-packages/product_details/__init__.py", line 31, in __init__
    json_dir=json_dir)
  File "/home/rehandalal/.virtualenvs/normandy/lib/python3.5/site-packages/product_details/storage.py", line 100, in __init__
    from product_details.models import ProductDetailsFile
  File "/home/rehandalal/.virtualenvs/normandy/lib/python3.5/site-packages/product_details/models.py", line 4, in <module>
    class ProductDetailsFile(models.Model):
  File "/home/rehandalal/.virtualenvs/normandy/lib/python3.5/site-packages/django/db/models/base.py", line 94, in __new__
    app_config = apps.get_containing_app_config(module)
  File "/home/rehandalal/.virtualenvs/normandy/lib/python3.5/site-packages/django/apps/registry.py", line 239, in get_containing_app_config
    self.check_apps_ready()
  File "/home/rehandalal/.virtualenvs/normandy/lib/python3.5/site-packages/django/apps/registry.py", line 124, in check_apps_ready
    raise AppRegistryNotReady("Apps aren't loaded yet.")
django.core.exceptions.AppRegistryNotReady: Apps aren't loaded yet.
```

I determined the offending commit by rolling back versions till I stopped seeing this. It works fine on v0.11.1 but not on v0.12.

**Comment by pmac** [comment]
That's really odd. What version of Django is this?

**Comment by rehandalal** [comment]
Django v1.9.11

**Comment by glogiotatidis** [comment]
I came across the same error. It appears that using `product_details` in my app's `__init__` caused the issue. Removing the lines from there and it all works. I'll make those lists populate lazily on first access.

https://github.com/mozmeao/snippets-service/blob/master/snippets/base/__init__.py#L8

**Comment by pmac** [comment]
This is now merged and released as version `0.13`.
